src/usr/share/gpgmailer/gpgmailmessage.py

```diff
@@ -75,10 +74,7 @@ def __init__(self):
                 'can be created.')
 
         self.saved = False
-        self.message = {}
-        self.message['body'] = None
-        self.message['attachments'] = []
-        self.message['subject'] = None
+        self.message = {'body': None, 'recipients': [], 'recipient_keys': [], 'attachments': [], 'subject': None}
```

Comment:
Was this an Intellij suggestion?

Reply:
yes. evidently dictionary literals are where it's at.